XamlControlsGallery/ControlPages/StandardUICommandPage.xaml

```diff
@@ -32,7 +32,7 @@
                 </Grid.RowDefinitions>
 
                 <TextBlock TextWrapping="Wrap" Margin="0,0,0,12">
-                    XamlUICommand allows the sharing of the UX associated with a command.
+                    StandardUICommand allows the sharing of the UX associated with a command.
                     In this instance we are using a StandardUICommand to quickly place 
                     the delete command in multiple controls. The XamlUICommand contains the icon, label, 
```

Comment:
Should this say "The **StandardUICommand** contains the icon, label, ..." instead of XamlUICommand? As a piece of meta feedback: I'm a little confused about the difference between the two concepts; it might be worthwhile to have some text somewhere that explicitly deliniates the two.

Reply:
I changed this line at the end to be specific about StandardUICommand. I think the very first thing you see at the top of this page is "A set of built-in XamlUICommands representing commonly used commands". I think this disambiguates well for those looking at the actual app.